server/app.js

```diff
@@ -1,17 +1,19 @@
+require('dotenv').config()
```

Comment:
Ada issue [codacy](https://app.codacy.com/gh/bellshade/Javascript/pullRequest?prid=9041083), harus pake doublequote sama semicolon

```suggestion
require("dotenv").config();
```

Reply:
> Ada issue [codacy](https://app.codacy.com/gh/bellshade/Javascript/pullRequest?prid=9041083), harus pake doublequote sama semicolon

sudah di edit dan di push silakan di cek kembali, terimakasih